scripts/build.py

```diff
@@ -0,0 +1,131 @@
+import os
+import shutil
+import re
+import sys
+import argparse
+from pathlib import Path
+
+
+def parse_args():
+    """Parse command line arguments."""
```

Comment:
Suggestion for python function comment header:

format it like this:

```
def dummy_func():
  """
  Create a new user.
  Line 2 of comment...
  And so on... 
  """
```

Also for any functions that take in parameters define the parameter, the type, and what it is. I know there are different python commenting schemas though so this isn't a requirement just something to keep in mind